config/charts/inferencepool/values.yaml

```diff
@@ -6,7 +6,14 @@ inferenceExtension:
     tag: main
     pullPolicy: Always
   extProcPort: 9002
-  env: []
+  # Environment variables for the EPP container.
```

Comment:
pls move this to https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/config/charts/inferencepool/templates/epp-deployment.yaml and remove the flag we set at https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/config/charts/inferencepool/templates/epp-deployment.yaml#L28

Reply:
Fixed

Reply:
One can set the flag explicitly in https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/config/charts/inferencepool/values.yaml#L34, right?